test/TestSetup.sol

```diff
@@ -140,19 +140,31 @@ contract TestSetup is Test, Configured {
 
         vm.warp(block.timestamp + delayModifier.txCooldown());
         uint256 txNonce = delayModifier.txNonce();
-        while (
-            delayModifier.txHash(txNonce)
-                != delayModifier.getTransactionHash(
-                    transaction.to, transaction.value, transaction.data, transaction.operation
-                )
-        ) {
+
+        Transaction memory unwrappedTransaction = this.unwrapTxData(transaction.data);
+        bytes32 txHash = delayModifier.getTransactionHash(
+            unwrappedTransaction.to,
+            unwrappedTransaction.value,
+            unwrappedTransaction.data,
+            unwrappedTransaction.operation
+        );
+
+        bool success;
+        while (delayModifier.txHash(txNonce) != txHash) {
             ++txNonce;
+            success = true;
```

Comment:
`success` should only be set when the tx hash is indeed found, which is the case as soon as the while loop is escaped. So you don't need this variable

Reply:
Ah nice catch! Then it'd be better to name it accordingly, for example: `txNonceIncremented` or `setTxNonce` or store the original `delayModifier.txNonce()` in a separate variable and compare it after the while loop to be even more explicit to the reader: `txNonce != currentTxNonce`